app/models/license.rb

```diff
@@ -45,15 +48,19 @@ class License < AbstractModel
   has_many :name_descriptions
```

Comment:
@nwilson-eol , @pellaea:
### Should I add a License `created_at` column?
Seems weird that there isn't one, but that (obviously) doesn't break anything.

(PS: 3 of the Licenses have a updated_at == nil. That also seems weird. 
2 will be changed (shortly after this PR is deployed) when they are deprecated. The third (publicdomain) won't be changed.)

Reply:
It does seem odd that there isn't a created_at.  I note that true of the nearby "interests" table as well.  Not sure what the thinking there was other than maybe trying to save some space?  At this point I'd rather it was all consistent with both a created_at and an updated_at.

Reply:
Thanks! I will:
- Add a `created_at` column as a migration in this PR.
- Insure that `created_at` and `updated_at` are present for all Licenses by -- after this PR is merged and Licenses are updated via the new UI -- making any needed fixes on the back end.